client/app/states/designer/blueprints/editor/editor.html

```diff
@@ -1,15 +1,14 @@
-<span ng-controller="BlueprintEditorController" ng-class="{'wait-cursor': loading}">
+<span ng-controller="BlueprintEditorController as evm" ng-class="{'wait-cursor': loading}">
```

Comment:
yep! totally wish em as directives/components, rid ourselves of that ng-controller in the html...   not sure its a straight forward task, hence the one of these days...